frontend/src/static/js/v_summary.js

```diff
@@ -849,9 +862,9 @@ window.vSummary = {
     },
   },
   created() {
+    this.processFileTypes();
```

Comment:
the reason why `processFileTypes` method is moved up is because `renderFilterHash` needs to access `fileTypes` array that is populated in the `processFileTypes` method.